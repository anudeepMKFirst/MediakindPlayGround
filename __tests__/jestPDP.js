import React from 'react';
import PDP from "./../pdp.youi.js";
import renderer from 'react-test-renderer';

it('renders correctly', () => {
  const tree = renderer
    .create(<PDP navigation={global.navigation}/>)
    .toJSON();
  expect(tree).toMatchSnapshot();
});