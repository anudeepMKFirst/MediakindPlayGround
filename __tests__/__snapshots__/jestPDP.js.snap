// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<Composition
  loadSync={true}
  onTimelinesLoaded={[Function]}
  source="PDP_Main"
>
  <TimelineRef
    name="Out"
    onAborted={[Function]}
    onCompleted={[Function]}
    onCompletedForward={[Function]}
    onCompletedReverse={[Function]}
    onCompositionDidLoad={[Function]}
    onPaused={[Function]}
    onPausedForward={[Function]}
    onPausedReverse={[Function]}
    onPlay={[Function]}
    onPlayForward={[Function]}
    onPlayReverse={[Function]}
    onStarted={[Function]}
    onStartedForward={[Function]}
    onStartedReverse={[Function]}
  />
  <TimelineRef
    name="In"
    onAborted={[Function]}
    onCompleted={[Function]}
    onCompletedForward={[Function]}
    onCompletedReverse={[Function]}
    onCompositionDidLoad={[Function]}
    onPaused={[Function]}
    onPausedForward={[Function]}
    onPausedReverse={[Function]}
    onPlay={[Function]}
    onPlayForward={[Function]}
    onPlayReverse={[Function]}
    onStarted={[Function]}
    onStartedForward={[Function]}
    onStartedReverse={[Function]}
  />
  <YiPushButtonView
    name="Btn-Play"
    onCompositionDidLoad={[Function]}
    onPress={[Function]}
    onTimelinesLoaded={[Function]}
  />
  <YiTextSceneNode
    name="MoreLikeThis-NotFound"
    visible={true}
  />
  <ViewRef
    name="ListAnchor"
    onCompositionDidLoad={[Function]}
    onTimelinesLoaded={[Function]}
  />
</Composition>
`;
